www-servers/nginx-unit/nginx-unit-1.12.0-r2.ebuild

```diff
@@ -17,7 +17,7 @@ SLOT="0"
 KEYWORDS="~amd64"
 MY_USE="perl python ruby"
 MY_USE_PHP="php5-6 php7-1 php7-2 php7-3"
-IUSE="${MY_USE} ${MY_USE_PHP}"
+IUSE="${MY_USE} ${MY_USE_PHP} ssl"
```

Comment:
You should also mention the addition of ssl USE somewhere in git summary or message box.

Reply:
The `ssl` flag was introduced in -r1, as mentioned in the commit message. This is another case of GitHub displaying the differences between a deleted, older ebuild and the latest one.